modules/model/shared/src/main/scala/gem/util/Location.scala

```diff
@@ -168,13 +169,13 @@ object Location {
       if (gapSize < BigDecimal.exact(1)) go(len + 1)
       else {
         // This is the existing start position as a BigDecimal.
-        val startBd = BigDecimal(start10, 0)
+        val startBd = BigDecimal(start10, 0, MathContext.UNLIMITED)
```

Comment:
Oh man, I was just looking at a PR to Cats to accommodate changes here in 2.13. What a mess.

Reply:
Yup, saw your comment over there. The idea had occurred to me of a difference in behavior in `BigDecimal` or `BigInt` but I was dismissing it as implausible. Until I googled "scala 2.13 bigdecimal" and found out about the whole issue.

The good news is that the test now passes with this "workaround".

Reply:
👍